src/main/java/ch/saunah/saunahbackend/dto/BookingPriceResponse.java

```diff
@@ -14,8 +14,7 @@ public class BookingPriceResponse {
     private double depositPrice;
     private double handTowelPrice;
     private double woodPrice;
-    private double discount;
-    private String discountDescription;
+    private double hourlyRate;
```

Comment:
This property is not needed. The hourly rate is the property `price` in the sauna. The hourly rate is defined individually for every sauna in the `price` property.